Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransporter.cs

```diff
@@ -545,7 +599,7 @@ private SingleFileTransferResult NoSourceOperation(BatchContext context, Source
         switch (_batchContext.Source.Action)
         {
             case SourceAction.Error:
-                _logger.NotifyError(context, msg, new FileNotFoundException());
+                _logger.NotifyError(context, msg, new ArgumentException(msg));
                 return new SingleFileTransferResult { Success = false, ErrorMessages = { msg } };
             case SourceAction.Info:
                 _logger.NotifyInformation(context, msg);

```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/267e98ab232f82b7a9f5fa1cee119a75ec7687d5/Frends.SFTP.DownloadFiles/Frends.SFTP.DownloadFiles/Definitions/FileTransporter.cs#L627
This method is not used, so it can be removed.